Sources/Core/GTMSessionFetcherService.m

```diff
@@ -36,6 +47,16 @@ - (BOOL)beginFetchMayDelay:(BOOL)mayDelay
 
 @interface GTMSessionFetcherService ()
 
+// If YES, |userAgentProvider| will default to |GTMStandardUserAgentProvider| for new instances of
+// this class. The User-Agent string will be calculated asynchronously off the calling thread and
+// cached.
+//
+// Otherwise, |userAgentProvider| will default to a |GTMUserAgentStringProvider| containing
+// |GTMFetcherStandardUserAgentString()| fetched on the calling thread (which can block).
+//
+// Defaults to NO.
+@property(class, atomic, assign) BOOL useStandardUserAgentProvider;
```

Comment:
Since this isn't public, why provide it?  Doesn't it also mean the only thing ever used is the `NO` code paths?